std_msgs/Header header
mav_trajectory_generation_ros2/PolynomialSegment[] segments